dask/dataframe/io/tests/test_csv.py

```diff
@@ -28,6 +28,13 @@
 from dask.utils import filetexts, filetext, tmpfile, tmpdir
 from fsspec.compression import compr
 
+try:
+    import distributed
+    from distributed import Client, wait
+    from distributed.utils_test import cluster, loop  # noqa: F401
+except (ImportError, SyntaxError):
+    distributed = None
```

Comment:
It is common to test tokenization by verifying that the name of two differently created collections is different, rather than engaging `distributed`.

Also, when we do engage distributed it's generally much nicer to use `gen_cluster` if possible.  It's faster, cleans up better, and easier to debug.

Reply:
Ah ok thanks for that :) I'll change the test to just check that the name is different.